README.md

```diff
@@ -2,7 +2,7 @@
 
 # fzf.fish 🔍🐟
 
-[![latest release badge][]][releases] [![fish version badge][]](#installation) [![awesome badge][]][awesome fish]
+[![latest release badge][]][releases] [![build status badge][]][actions] [![awesome badge][]][awesome fish]
```

Comment:
I figured seeing the build status is more helpful and assuring than the fish version. People likely have the latest fish anyway and the installation instructions make it clear what version is necessary.